FV3/makefile

```diff
@@ -191,6 +191,7 @@ clean:
 	(cd io                    && make clean)
 	(cd atmos_cubed_sphere    && make clean)
 	(cd cpl                   && make clean)
+	(cd coarse_graining       && make clean)
 	$(RM) -f $(FV3_EXE) $(FV3CAP_LIB) *.o *.mod *.i90 *.lst depend
 
 clean_no_dycore:
```

Comment:
For completeness should you add it under this rule too?

Reply:
Good catch, added. I think we don't use the _no_dycore items anymore but it's good to be complete.